include/inria_wbc/controllers/controller.hpp

```diff
@@ -91,10 +107,8 @@ namespace inria_wbc {
 
             std::shared_ptr<tsid::tasks::TaskComEquality> make_com_task(const std::string& name, double kp) const;
             std::shared_ptr<tsid::tasks::TaskJointPosture> make_posture_task(const std::string& name, double kp) const;
-            std::shared_ptr<tsid::tasks::TaskSE3Equality> make_torso_task(const std::string& name, const std::string& frame_name, double kp) const;
-            std::shared_ptr<tsid::tasks::TaskSE3Equality> make_floatingb_task(const std::string& name, const std::string& joint_name, double kp) const;
-            std::shared_ptr<tsid::tasks::TaskSE3Equality> make_hand_task(const std::string& name, const std::string& joint_name, double kp) const;
-            std::shared_ptr<tsid::tasks::TaskSE3Equality> make_foot_task(const std::string& name, const std::string& joint_name, double kp) const;
+            std::shared_ptr<tsid::tasks::TaskSE3Equality> make_se3_frame_task(const std::string& name, const std::string& frame_name, double kp, const mask::mask6& mask = mask::all) const;
+            std::shared_ptr<tsid::tasks::TaskSE3Equality> make_se3_task(const std::string& name, const std::string& joint_name, double kp, const mask::mask6& mask = mask::all) const;
```

Comment:
maybe `make_se3_joint_task`?

Reply:
yes, seems more clearand will be closer to the make_se3_frame nomeclature